src/supertux/main.cpp

```diff
@@ -207,18 +208,55 @@ class PhysfsSubsystem
     }
     else
     {
-      std::string physfs_userdir = PHYSFS_getUserDir();
+		userdir = PHYSFS_getPrefDir("SuperTux","supertux2");
```

Comment:
That's right-I now remember why I didn't merge it: the physfs version is
better, but it is not in the latest release (it is only in trunk).  Also,
let's still wait to merge it-the SDL version does not quite do what we want
either.

On Wed, Mar 23, 2016 at 12:34 PM, Tobias Markus <notifications@github.com>
wrote:

> In src/supertux/main.cpp
> <https://github.com/SuperTux/supertux/pull/294#discussion_r57202750>:
>
> > @@ -207,18 +208,55 @@ class PhysfsSubsystem
> >      }
> >      else
> >      {
> > -      std::string physfs_userdir = PHYSFS_getUserDir();
> > +		userdir = PHYSFS_getPrefDir("SuperTux","supertux2");
>
> This should probably be https://wiki.libsdl.org/SDL_GetPrefPath
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly or view it on GitHub
> <https://github.com/SuperTux/supertux/pull/294/files/09aa84e7136547908b813c22ecf1dde8f5f6b52f#r57202750>
>